src/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
I don't think we should include this file because it is not relevant and might lead to confusion later on. Maybe you can exclude it from this PR?

Reply:
I have removed the unnecessary files and information @MandelaK. Please review again.